src/Compilers/CSharp/Test/Syntax/Parsing/RawInterpolatedStringLiteralCompilingTests.cs

```diff
@@ -0,0 +1,1757 @@
+    }
+
+    [Fact]
+    public void MultiLineCase33()
+    {
+        RenderAndVerify(@"
+System.Console.Write(
+    $""""""
+  a
+    {42}
+    b
+    {43}
+    c
+    """""");",
+                // (4,1): error CS9103: Line does not start with the same whitespace as the last line of the raw string literal
+                //   a
+                Diagnostic(ErrorCode.ERR_LineDoesNotStartWithSameWhitespace, "  ").WithLocation(4, 1));
+    }
+
+    [Fact]
+    public void MultiLineCase34()
+    {
+        RenderAndVerify(@"
+System.Console.Write(
+    $""""""
+    a
+  {42}
+    b
+    {43}
+    c
+    """""");",
+                // (5,1): error CS9103: Line does not start with the same whitespace as the last line of the raw string literal
+                //   {42}
+                Diagnostic(ErrorCode.ERR_LineDoesNotStartWithSameWhitespace, "  ").WithLocation(5, 1));
+    }
+
+    [Fact]
+    public void MultiLineCase35()
+    {
+        RenderAndVerify(@"
+System.Console.Write(
+    $""""""
+    a
+    {42}
+  b
+    {43}
+    c
+    """""");",
+                // (6,1): error CS9103: Line does not start with the same whitespace as the last line of the raw string literal
+                //   b
+                Diagnostic(ErrorCode.ERR_LineDoesNotStartWithSameWhitespace, "  ").WithLocation(6, 1));
+    }
+
+    [Fact]
+    public void MultiLineCase36()
+    {
+        RenderAndVerify(@"
+System.Console.Write(
+    $""""""
+    a
+    {42}
+    b
+  {43}
+    c
+    """""");",
+                // (7,1): error CS9103: Line does not start with the same whitespace as the last line of the raw string literal
+                //   {43}
+                Diagnostic(ErrorCode.ERR_LineDoesNotStartWithSameWhitespace, "  ").WithLocation(7, 1));
+    }
+
+    [Fact]
+    public void MultiLineCase37()
+    {
+        RenderAndVerify(@"
+System.Console.Write(
+    $""""""
+    a
+    {42}
+    b
+    {43}
+  c
+    """""");",
+                // (8,1): error CS9103: Line does not start with the same whitespace as the last line of the raw string literal
+                //   c
+                Diagnostic(ErrorCode.ERR_LineDoesNotStartWithSameWhitespace, "  ").WithLocation(8, 1));
+    }
+
+    [Fact]
+    public void MultiLineCase38()
+    {
+        RenderAndVerify(@"
+System.Console.Write(
+    $""""""
+    a
+  aa
+    {42}
+    b
+    {43}
+    c
+    """""");",
+                // (5,1): error CS9103: Line does not start with the same whitespace as the last line of the raw string literal
+                //   aa
+                Diagnostic(ErrorCode.ERR_LineDoesNotStartWithSameWhitespace, "  ").WithLocation(5, 1));
+    }
+
+    [Fact]
+    public void MultiLineCase39()
+    {
+        RenderAndVerify(@"
+System.Console.Write(
+    $""""""
+    a
+    {42}
+  {42}
+    b
+    {43}
+    c
+    """""");",
+                // (6,1): error CS9103: Line does not start with the same whitespace as the last line of the raw string literal
```

Comment:
nit: it's debatable what "last line of the raw string literal" means. Maybe "terminating line" or "closing line"?

Reply:
I like that!

Reply:
done.  i used "closing line".